src/Redux/books/books.js

```diff
@@ -0,0 +1,23 @@
+const BOOK_ADDED = './react-bookstore/bookreducer/BOOK_ADDED';
+const BOOK_DELETED = './react-bookstore/bookreducer/BOOK_DELETED';
+
+export const bookAdded = (book) => ({
+  type: BOOK_ADDED,
+  payload: book,
+});
+
+export const deleteBook = (id) => ({
+  type: BOOK_DELETED,
+  payload: id,
+});
+
```

Comment:
[OPTIONAL] Your code looks great, well done! I would suggest using the same naming style for similar variables and functions, this way it would be easier to identify when several things are related, for example, if you have `bookAdded` then you can go `bookDeleted bookEdited bookCreated` and if you have `deleteBook` then you can go `addBook createBook editBook`.